packages/esm-patient-search-app/src/patient-search-result/patient-search-result.component.tsx

```diff
@@ -27,7 +46,7 @@ const PatientSearchResults: React.FC<PatientSearchResultsProps> = ({ patients, h
           },
         ],
         gender: patient.person.gender,
-        birthDate: patient.person.birthdate,
+        birthDate: gregToEth(patient.person.birthdate),
```

Comment:
This should be configurable to be able to handle both the Ethiopian date and the Gregorian date formats